libs/host/Configuration/Options.cs

```diff
@@ -663,6 +663,14 @@ private IAuthenticationSettings GetAuthenticationSettings(ILogger logger = null)
                 case GarnetAuthenticationMode.AclWithAad:
                     var aadAuthSettings = new AadAuthenticationSettings(AuthorizedAadApplicationIds?.Split(','), AadAudiences?.Split(','), AadIssuers?.Split(','), IssuerSigningTokenProvider.Create(AadAuthority, logger), AadValidateUsername.GetValueOrDefault());
                     return new AclAuthenticationAadSettings(AclFile, Password, aadAuthSettings);
+                case GarnetAuthenticationMode.Custom:
```

Comment:
maybe we could default this to NoAuth and print a warning message. The caller is responsible for providing authentication settings. It might be better than passing it as a parameter.

Reply:
Updated